e2e/operator_installation_test.go

```diff
@@ -55,6 +55,13 @@ var _ = Describe("Operator Installation Tests", func() {
 		})
 
 		It("should install into current namespace", func() {
+			if !RunsAgainstOpenshift {
+				Skip("OpenShift specific test assuming current namespace is set and oc available")
```

Comment:
Add some notes around "skip" explaining how this is covered else where or similar if it's no a pure Openshift test

Reply:
Where do you think we should add that? In the message itself, docs or ...?

Reply:
A comment to the Skip line would do

Reply:
598cd32